mne/viz/_brain/_brain.py

```diff
@@ -1202,59 +1215,84 @@ def _save_movie_noname(self):
         return self.save_movie(None)
 
     def _configure_tool_bar(self):
-        self.actions["screenshot"] = self.tool_bar.addAction(
-            self.icons["screenshot"],
-            "Take a screenshot",
-            self.plotter._qt_screenshot
-        )
-        self.actions["movie"] = self.tool_bar.addAction(
-            self.icons["movie"],
-            "Save movie...",
-            self._save_movie_noname,
-        )
-        self.actions["visibility"] = self.tool_bar.addAction(
-            self.icons["visibility_on"],
-            "Toggle Visibility",
-            self.toggle_interface
-        )
-        self.actions["play"] = self.tool_bar.addAction(
-            self.icons["play"],
-            "Play/Pause",
-            self.toggle_playback
-        )
-        self.actions["reset"] = self.tool_bar.addAction(
-            self.icons["reset"],
-            "Reset",
-            self.reset
-        )
-        self.actions["scale"] = self.tool_bar.addAction(
-            self.icons["scale"],
-            "Auto-Scale",
-            self.apply_auto_scaling
-        )
-        self.actions["restore"] = self.tool_bar.addAction(
-            self.icons["restore"],
-            "Restore scaling",
-            self.restore_user_scaling
-        )
-        self.actions["clear"] = self.tool_bar.addAction(
-            self.icons["clear"],
-            "Clear traces",
-            self.clear_glyphs
-        )
-        self.actions["help"] = self.tool_bar.addAction(
-            self.icons["help"],
-            "Help",
-            self.help
-        )
+        if self.notebook:
+            from IPython import display
+            from ipywidgets import HBox, Button
+            self.actions["visibility"] = Button(
+                description="Toggle Visibility",
+            )
+            self.actions["visibility"].on_click(
+                lambda x: self.toggle_interface())
+            self.actions["reset"] = Button(description="Reset")
+            self.actions["reset"].on_click(lambda x: self.reset())
+            self.actions["scale"] = Button(description="Auto-Scale")
+            self.actions["scale"].on_click(lambda x: self.apply_auto_scaling())
+            self.actions["restore"] = Button(description="Restore scaling")
+            self.actions["restore"].on_click(
+                lambda x: self.restore_user_scaling())
+            self.actions["clear"] = Button(description="Clear traces")
+            self.actions["clear"].on_click(lambda x: self.clear_glyphs())
```

Comment:
This is pretty non-DRY -- any new action requires adding something in two places now.

How about a pattern more like:
```
    self.initialize_actions()
    self.add_action('Reset', self.reset)
    ...
    if not self.notebook:
        self.add_action('Screenshot', ...)
    ...
    self.layout_actions()

...

def initialize_actions(self):
    if not self.notebook:
        self._load_icons()
        ...
    else:
        ...

...

def self.add_action(name, func):
    if self.notebook:
        self.actions[name] = Button(description=name)
        self.actions[name].on_click(lambda x: func())
    else:
        self.actions[name] = self.tool_bar.addAction(...)
```
It makes immediately clear what's available in both backends versus one or the other. And to add a new interaction you just add it in one place.

Reply:
It should be better after af7f988 but the icons' order in the tool bar has changed.

Reply:
> It should be better after af7f988 but the icons' order in the tool bar has changed.

I would not change the order. Instead you can do:
```
self._add_action(..., notebook=False)
```
and in `_add_action` only add it to the notebook when `notebook=True` (default). Then the order can be whatever you want (i.e., as it was in `master`) and it's still clear which ones are omitted from the notebook.

Reply:
... or instead of adding `notebook=True` kwarg, just write:
```
if not self.notebook:
    self._add_action(...)
```
inline as needed. But it seems slightly nicer and more compact just to have a `notebook` kwarg in the `_add_action` as suggested above

Reply:
The order is restored now :+1: